CMakeLists.txt

```
@@ -31,6 +31,14 @@ if(CHIRA_COMPILER_ID MATCHES "clang")
     endif()
 endif()
 
+# Detect macOS so we can make needed changes
+if(${CMAKE_SYSTEM} MATCHES "Darwin")
+	add_definitions(-DMACOSX)
+    # Include CoreFoundation Library as we absolutely need this
```

Definition isn't necessary and `add_definitions` shouldn't be used either way
```suggestion
	# Include CoreFoundation Library as we absolutely need this
```

---

~~blame STJr./SRB2 for this atrocity~~ yeah yeah that's entirely correct haha that's not a good thing to do